### PR TITLE
adds beginning css

### PR DESCRIPTION
**Pull request by he8079**
more info on flexbox for the menu and footer needed.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,44 +4,111 @@
...
+    <li class="toggle"><i class="fa-solid fa-bars"></i></li>
+</ul>
+</div>
     </nav>
 </header>    
 <main>
-    <div> 
-        <h1>Testimonials</h1>
-        <p>Testimonial from a client</p>
+   <div class="slideshow-container">
```
Comment (axamei) [inline index.html:43]: missing an h1 on this page. you can hide it visually if you don't want it displayed, but it needs to exist. see https://www.a11yproject.com/posts/how-to-hide-content/

File: index.html [hunk 2]
```diff
@@ -4,44 +4,111 @@
...
-        <h2>Copyright</h2>
-        <p>2022</p>
+   
+    <div class="estimate">
+        <p>*Always free estimates on services*</p>
     </div>
     <div>
-        <h2>Contact Us</h2>
-        <p>moorestudentsuccess@gmail.com</p>
+        <table class="indextable">
```
Comment (axamei) [inline index.html]: tables should only be used as a data table / comparing data. use divs and flexbox for this layout

**Review by axamei — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (he8079) [reply index.html:43]: Fixed h1 on index page.

File: index.html (hunk 2 above)
Comment (he8079) [reply index.html]: The table is only a placeholder and I will be changing it this week. I wanted to try making a basic table to learn how. I never intended it to be permanent.

Comment (he8079) [reply index.html]: table removed and updated with div boxes.

**Review by axamei — APPROVED** [review]
👍

**Pull request merged**